assets/src/edit-story/components/panels/design/preset/useKeyboardNavigation.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { useKeyDownEffect } from '../../../../../design-system/components/keyboard';
+import {
+  COLOR_PRESETS_PER_ROW,
+  STYLE_PRESETS_PER_ROW,
+} from '../../../../constants';
+
+function useKeyboardNavigation({
```

Comment:
Have you checked if you can use the existing `useRovingTabIndex` here instead?

Reply:
Yes, tried using that first, didn't work. Will check again just in case.